models/baseModels/tests/testLead.spec.ts

```diff
@@ -0,0 +1,125 @@
+import test from 'tape';
+import { closeTestFyo, getTestFyo, setupTestFyo } from 'tests/helpers';
+import { ModelNameEnum } from 'models/types';
+import { Lead } from '../Lead/Lead';
+import { Party } from '../Party/Party';
+
+const fyo = getTestFyo();
+setupTestFyo(fyo, __filename);
+
+const leadData = {
+  name: 'name2',
+  status: 'Open',
+  email: 'sample@gmail.com',
+  mobile: '1231233545',
+};
+
+const itemData: { name: string; rate: number } = {
+  name: 'Pen',
+  rate: 100,
+};
+
+test('create test docs for Lead', async (t) => {
+  await fyo.doc.getNewDoc(ModelNameEnum.Lead, leadData).sync();
+
+  t.ok(
+    fyo.db.exists(ModelNameEnum.Lead, leadData.name),
+    `dummy item ${leadData.name} exists`
+  );
+});
```

Comment:
need to change the name of the test to something suitable. need to change the message that checks documents existence